test/functional/provider/define_spec.lua

```diff
@@ -333,6 +333,39 @@ local function function_specs_for(fn, sync, first_arg_factory, init)
           runx(sync, handler, on_setup)
         end)
       end)
+
+      describe('with range', function()
```

Comment:
if a `describe` just contains a single `it` you can just as well merge them to `it('with range', function() ...`. (or is there an underlying reason other than this file being old? ping @tarruda)